README.md

```diff
@@ -39,8 +32,6 @@ The following environment variables are supported in the default configuration:
 |APM_SECRETTOKEN    | Secret token, if required. |
 |APM_QUERYLOG       | `true` or `false` defaults to 'true'. Set to `false` to completely disable query logging, or to `auto` if you would like to use the threshold feature. |
 |APM_THRESHOLD      | Query threshold in milliseconds, defaults to `200`. If a query takes longer then 200ms, we enable the query log. Make sure you set `APM_QUERYLOG=auto`. |
-|APM_BACKTRACEDEPTH | Defaults to `25`. Depth of backtrace in query span. |
-|APM_RENDERSOURCE   | Defaults to `true`. Include source code in query span. |
```

Comment:
I removed these two properties because we are not using them yet.

Reply:
It's still in https://github.com/arkaitzgarro/elastic-apm-laravel/blob/71bf40baeb0ff03edeb8aedafbab8eafc1ba43c7/config/elastic-apm-laravel.php .

Reply:
`APM_BACKTRACEDEPTH` is used [by the agent](https://github.com/philkra/elastic-apm-php-agent/blob/1fdf7cf342d62603b7668d99e1fab7300c8ae2cf/src/Agent.php#L186), we should not remove it.